packages/styling/src/styles/DefaultFontStyles.ts

```diff
@@ -84,7 +84,7 @@ function _getFontBaseUrl(): string {
   let win = getWindow();
 
   // tslint:disable-next-line:no-string-literal no-any
```

Comment:
```suggestion
  // tslint:disable-next-line:no-any
```

Reply:
there's no reason to use a string literal here. it would still need the `any` type cast.

Reply:
Right so you should remove the lint rule override relating to string literals.

Reply:
oh doh! I misread :)